src/build.js

```diff
@@ -17,15 +17,18 @@ const walk = async dir => {
     const stat = fs.statSync(filePath)
 
     if (stat.isDirectory()) {
-      tree[propName] = await walk(filePath)
+      tree[propName] = await walk(filePath, wombatUrl)
     }
 
     if (stat.isFile()) {
-      const content = fs.readFileSync(filePath, 'utf8')
+      let content = fs.readFileSync(filePath, 'utf8')
       const extension = path.extname(filePath)
 
       switch(extension) {
         case '.json':
+          if(wombatUrl && content.includes('wombatUrl')) {
```

Comment:
To nie powinno byc ograniczone tylko do jsonow, w markdownie tez sa jakies obrazki linki itd.

Dodatkowo zrezygnowalbym ze sprawdzania czy ten `wombatUrl` wystepuj w tresci, bo jezeli regexp niczego nie zlapie, to nic nie zamieni, a tak leci dwa razy po tym samym, ale w sumie szukajac dwoch roznych stringow.

Reply:
Poprawione